### PR TITLE
Link and Path fields added

### PR DESCRIPTION
**Pull request by sadrasabouri**
#### Reference Issues/PRs
#99 
#### What does this implement/fix? Explain your changes.
Since we need to know what's the link of uploaded art in the nft storage we decided to return the link to the art in the when using `nft_storage_upload` method.
Same issue when saving a config/data/art file.

### CODE REVIEW

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]
@sadrasabouri Thanks for your efforts 💯 
JSON field shouldn't be dynamic (sometimes `message` sometimes `link`/`path`)
I suggest replacing `link` and `path` with `message`

**Comment by sadrasabouri** [comment]
> 

I think message field is necessary in some cases in which we return error message in `message` field and these error messages can not be in the `link`/`path` field.

File: samila/params.py [hunk 1]
```diff
@@ -28,6 +28,7 @@
 SEED_UPPER_BOUND = 2**20
 VALID_COLORS = list(dict(mcolors.BASE_COLORS, **mcolors.CSS4_COLORS).keys())
 NFT_STORAGE_API = "https://api.nft.storage/upload"
+NFT_STORAGE_LINK = "https://{}.ipfs.nftstorage.link/"
```
Comment (sepandhaghighi) [inline samila/params.py:31]: I suggest this format:
`https://ipfs.io/ipfs/{}`

File: samila/functions.py [hunk 2]
```diff
@@ -339,6 +342,7 @@ def save_data_file(g, file_adr):
     try:
         with open(file_adr, 'w') as fp:
             json.dump(data, fp)
+        result["message"] = os.path.dirname(os.path.abspath(file_adr))
```
Comment (sepandhaghighi) [inline samila/functions.py:345]: I think a direct file address (`os.path.abspath(file_adr)`) is better than a folder address. isn't?

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]

**Review by sepandhaghighi — APPROVED** [review]
LGTM 🚀

**Pull request merged**